code/zk-game/contracts/deploy/utils.ts

```diff
@@ -0,0 +1,177 @@
+import { Provider, Wallet } from 'zksync-ethers';
+import * as hre from 'hardhat';
+import { Deployer } from '@matterlabs/hardhat-zksync';
```

Comment:
I'd avoid using zksync-specific features like the `Deployer` class. Now we can use ethers `contractFactory` and deploy as on EVM. Here is an example: https://github.com/matter-labs/paymaster-examples/blob/main/contracts/deploy/utils.ts